keystone_api/apps/allocations/tasks.py

```diff
@@ -50,7 +50,7 @@ def update_limit_for_account(account_name: str, cluster: Cluster) -> None:
         return
 
     # Base query for approved Allocations under the account on this cluster
-    acct_alloc_query = Allocation.objects.filter(request__group=account, cluster=cluster, request__approved=True)
+    acct_alloc_query = Allocation.objects.filter(request__group=account, cluster=cluster, request__approved__isnull=False)
```

Comment:
`request__approved` is a date, not a boolean value. I believe it is only set when the request is approved, so this yields the correct behavior.